client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -278,7 +298,7 @@ export const siteSetupFlow: Flow = {
 				}
 
 				case 'vertical': {
-					return navigate( 'intent' );
+					return navigate( intentCaptureStep );
 				}
```

Comment:
Under the scenario where both GC + V13N are enabled, it should:
1. Check if the goal `import` is selected, and route to Import flow `import`.
2. Check if the intent is `write` or `sell`, then route to Write/Sell flow `options`.
3. Else if the intent is `build`, then route to Build flow `designSetup`.

Reply:
What about the `difm` goal/intent was selected? Do we still go through `vertical` step?  And then, only after, exit out to `/start/website-design-services/?siteSlug=${ siteSlug }`? Or we skip the `vertical` step then and exit out directly from `goals` capture screen?